source/mainnet/smart-contracts/tutorials/wCCD/wCCD-front-end-set-up.rst

```diff
@@ -4,6 +4,346 @@
 Setting up the front-end
 ========================
 
-Coming soon.
+.. note::
+
+    Before you start with part 3 of this tutorial, make sure you have:
+
+    - access to the `Chrome <https://www.google.com/chrome/>`_ web browser on your laptop or computer
+
+    - `node.js <https://nodejs.org/en/download/>`_ and `yarn <https://yarnpkg.com/getting-started/install>`_ installed
+
```

Comment:
Added line that users have to download `git` before starting the tutorial (to be able to clone a repo from Github).

Since the cloning commands are changed to `HTTPS`, I did not mention ``ssh-client``.